packages/nimble-components/src/select/styles.ts

```diff
@@ -29,6 +30,25 @@ export const styles = css`
          error icon, and dropdown arrow because they are not "interactive" i.e. part of the tab order */ ''
     }
 
+    ::slotted(nimble-list-option-group.show-top-separator)::before,
```

Comment:
Should all of the separator styling be in the list-option-group styles file? It seems like that would be the most reusable place for when grouping is added to other components, such as the combobox.